docs/installation.rst

```diff
@@ -2,45 +2,57 @@
 Installation
 ============
 
-**Starting from scratch**
+The following instructions will allow you to install pysatMissions.
 
-----
+Prerequisites
+-------------
 
-Python and associated packages for science are freely available. Convenient science python package setups are available from https://www.python.org/ and `Anaconda <https://www.anaconda.com/distribution/>`_. Anaconda also includes a developer environment. Core science packages such as numpy, scipy, pandas and many others may also be installed directly via pip or your favorite package manager.
+.. image:: figures/poweredbypysat.png
+    :width: 150px
+    :align: right
+    :alt: powered by pysat Logo, blue planet with orbiting python and the logo superimposed
 
-For educational users, developer environments from `Jet Brains <https://www.jetbrains.com/student/>`_ are available for free.
 
+pysatMissions uses common Python modules, as well as modules developed by
+and for the Space Physics community.  This module officially supports
+Python 3.7+ and pysat 3.0.0+.
 
-**pysatMissions**
+ ================ =================
+ Common modules   Community modules
+ ================ =================
+  numpy            pysat>=3.0.0
+  pandas           aacgmv2
+                   apexpy
+                   pyEphem
+                   sgp4
+ ================ =================
```

Comment:
this bit does not format since the first column is blank.  We may need a new format, or just swap the columns

Reply:
Are these going to be swapped?

Reply:
I'm fine with swapping these